po/ar.po

```diff
@@ -1495,12 +1566,19 @@ msgid ""
 "offer a solution, e.g., by showing people where to find the relevant "
 "information in the standard library."
 msgstr ""
+"دع الناس يحلون التمارين بأنفسهم أو في مجموعات صغيرة. نحن عادة نقضي 30-45 "
```

Comment:
I think here we should replace people with participants in the english translation.
```suggestion
"دع المشاركون يحلون التمارين بأنفسهم أو في مجموعات صغيرة. نحن عادة نقضي 30-45 "
```